### PR TITLE
Pallet ibc update

### PR DESCRIPTION
**Pull request by Wizdave97**
## Issue
*Please replace this line with either a link to the ClickUp issue, or a reference to the GitHub issue.*

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by seunlanlege** [comment]
conflicts

**Comment by Wizdave97** [comment]
> conflicts

Fixing them

**Comment by Wizdave97** [comment]
> conflicts

Fixing them

File: frame/ibc/ibc-rpc/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ibc-proto = { git = "https://github.com/ComposableFi/ibc-rs", rev = "a189a3b796cbb84807061180a4ad67c4a1ded8d0", default-features = false }
+ibc-runtime-api = { path = "../ibc-runtime-api" }
+sc-chain-spec = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+sp-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+sp-trie = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24" }
+tendermint-proto = { git = "https://github.com/composableFi/tendermint-rs", rev = "5a74e0f8da4d3dab83cc04b5f1363b018cf3d9e8" }
```
Comment (seunlanlege) [inline frame/ibc/ibc-rpc/Cargo.toml]: this file shouldn't exist

**Pull request merged**